THIRD_PARTY_NOTICES.md

```diff
@@ -7,26 +7,461 @@ below.
 In the event that a required notice is missing or incorrect, please notify us
 by e-mailing [open-source@newrelic.com](mailto:open-source@newrelic.com).
 
-For any licenses that require the disclosure of source code, the source code
-can be found at https://github.com/newrelic/node-newrelic.
+For any licenses that require the disclosure of source
+code, the source code can be found at [https://github.com/newrelic/node-newrelic](https://github.com/newrelic/node-newrelic).
 
-## Contents
+## Content
 
+**[dependencies](#dependencies)**
+
+* [@grpc/proto-loader](#grpcproto-loader)
+* [@newrelic/aws-sdk](#newrelicaws-sdk)
+* [@newrelic/koa](#newrelickoa)
+* [@newrelic/superagent](#newrelicsuperagent)
+* [@tyriar/fibonacci-heap](#tyriarfibonacci-heap)
 * [async](#async)
-* [concat-stream](#concat-steam)
+* [concat-stream](#concat-stream)
 * [https-proxy-agent](#https-proxy-agent)
 * [json-stringify-safe](#json-stringify-safe)
 * [readable-stream](#readable-stream)
 * [semver](#semver)
-* [@grpc/grpc-js](#@grpc/grpc-js)
```

Comment:
i see proto-loader got moved above but i don't see grpc-js? you might double-check your package-lock is in good shape.